week-1-ts-ecosystem/w01e01-typescript-basics/nodemon.json

```diff
@@ -0,0 +1,6 @@
+{
+  "watch": ["src"],
+  "ext": ".ts,.js",
+  "ignore": [],
+  "exec": "ts-node ./src/main.ts"
+}
```

Comment:
W tej configuracji nie ma nic złego, jednak czy kojarzysz że `nodemon` orientuje się że masz `ts-node` zainstalowanego i "out of the box" możesz napsiać w skrypcie:

`nodemon ./src/main.ts`

i będzie działało.

Reply:
Nie wiedziałem, dzięki